doc/base/classes.xml

```diff
@@ -15835,13 +15841,33 @@ Example: (content-length:12), (Content-Type:application/json; charset=UTF-8)
 			<argument index="0" name="device" type="int">
 			</argument>
 			<description>
-			Returns the duration of the current vibration effect in seconds.
+			If the device has an accelerometer, this will return the movement.
```

Comment:
Hmm, I see. But this commit is literally just me running doctool on a freshly build godot. Will fix it though.